doc/raspberrypi-64.md

```diff
@@ -1,7 +1,7 @@
-# Raspberry Pi 3 Model Series
+# Raspberry Pi 3 and 4 Model Series
 
 EMLinux for Raspberry Pi is __not__ officially supported. However Raspberry Pi is a popular device so it would be nice to test and evaluate EMLinux on it.
-This document describes how to setup EMLinux for the Raspberry Pi 3 model series. The current description is about Raspberry Pi 3 model B+ only.
+This document describes how to setup EMLinux for the Raspberry Pi 3 and 4 model series. The current description is concerning Raspberry Pi 3 model B+ and Raspberry Pi 4 model B.
```

Comment:
I'm not so particular about this remark, but have a trivial advice to you. 

In general, reviewers would like less numbers of changes when a PR is received because they can finish the review with less effort. You changed `about` to `concerning`. I know they have the same meaning and `concerning` is more formal expression, so I think it's no problem about this change. However some reviewers don't like such a change an explicit reason is not present. You can avoid it from now on,

No need to add any changes for this remark.